apps/prairielearn/src/pages/jobSequence/jobSequence.html.ts

```diff
@@ -13,31 +12,28 @@ export function JobSequence({
   resLocals: Record<string, any>;
   job_sequence: JobSequenceWithTokens;
 }) {
-  return html`
-    <!doctype html>
-    <html lang="en">
-      <head>
-        ${HeadContents({
-          resLocals,
-          pageTitle: `${job_sequence.description} #${job_sequence.number}`,
-        })}
-        ${compiledScriptTag('jobSequenceClient.ts')}
-      </head>
-      <body>
-        ${Navbar({ resLocals })}
-        <main id="content" class="container-fluid">
-          <h1 class="sr-only">Job Sequence</h1>
-          <div class="row">
-            <div class="col-12">
-              <a class="btn btn-primary mb-4" href="javascript:history.back();">
-                <i class="fa fa-arrow-left" aria-hidden="true"></i>
-                Back to previous page
-              </a>
-            </div>
-          </div>
-          ${JobSequenceResults({ course: resLocals.course, jobSequence: job_sequence })}
-        </main>
-      </body>
-    </html>
-  `.toString();
+  return PageLayout({
+    resLocals,
+    pageTitle: `${job_sequence.description} #${job_sequence.number}`,
+    navContext: {
+      type: 'plain',
+      page: 'admin',
+    },
+    options: {
+      fullWidth: true,
+    },
+    headContent: html`${compiledScriptTag('jobSequenceClient.ts')}`,
```

Comment:
Nit, you can do this instead, which is a little more concise (`HtmlSafeString[]` is compatible with `HtmlValue`:

```suggestion
    headContent: [compiledScriptTag('jobSequenceClient.ts')],
```